lsp/client-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "nls",
-  "description": "Nickel Language Server",
+  "name": "vscode-nickel",
+  "description": "Nickel Language",
```

Comment:
This will be the title (name) and description in the vscode marketplace.

Reply:
After a quick look at the VSCode market place, many extensions (not necessarily languages) seem to follow the `vscode-xxx` naming scheme, many language extensions don't, but that name sounds reasonable :+1:

Reply:
Ah, there's also `displayName` as `name` can't contain spaces. Do we want something like  "Nickel Language" instead? And add some more details in "description“?